MAUI/Cartesian-Charts/Get-the-touch-position-in-chart.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: Get the touch position in Syncfusion SfCartesianChart
+description: Get the touch position in SfCartesianChart
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# Get the touch position in SfCartesianChart
+
+ChartInteractiveBehavior provides the following override methods to get the x and y positions when touch the [`SfCartesianChart`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1).
+
+* [`OnTouchUp`]() - Called when a user lifts their finger or release their touch input from the Chart area. 
+* [`OnTouchMove`]() - Called when a user's finger or touch input device is in contact within the Chart area and moves across it's surface.
+* [`OnTouchDown`]() -  Called when the user made the initial contact of a user's finger or touch input device with the Chart Area.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfCartesianChart>
+        .........
+
+        <chart:SfCartesianChart.InteractiveBehavior>
+                <local:ChartInteractiveExt></local:ChartInteractiveExt>
+        </chart:SfCartesianChart.InteractiveBehavior>
+
+    </chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% endtabs %}
+
+{% tabs %}
+
+{% highlight c# %}
+
+    public class ChartInteractiveExt: ChartInteractiveBehavior
+    {
+        protected override void OnTouchDown(ChartBase chart, float pointX, float pointY)
+        {
+            base.OnTouchDown(chart, pointX, pointY);
+        }
+
+        protected override void OnTouchMove(ChartBase chart, float pointX, float pointY)
+        {
+            base.OnTouchMove(chart, pointX, pointY);
+        }
+
+        protected override void OnTouchUp(ChartBase chart, float pointX, float pointY)
+        {
+            base.OnTouchUp(chart, pointX, pointY);
+        }
+    }
+
+{% endhighlight  %}
+
+{% endtabs %}
+
+# See also
+
```

Comment:
i am not ask to add this now, we will add once we published. remove see also.

Reply:
I removed see also section